plugins/lookup/tss.py

```diff
@@ -157,10 +165,32 @@
   tasks:
       - ansible.builtin.debug:
           msg: the password is {{ secret_password }}
+
+- hosts: localhost
```

Comment:
A comment explaining what happens here would be useful. Especially how `file_download_path` and `fetch_attachments` are related to the private keys that are shown in `debug`.

Reply:
Comment added